integreat_cms/nominatim_api/nominatim_api_client.py

```diff
@@ -71,7 +71,7 @@ def search(
             raise RuntimeError(
                 "You can either specify query_str or pass additional keyword arguments, not both."
             )
-        if "street" in query_dict:
+        if "street" in query_dict and query_dict["street"]:
```

Comment:
```suggestion
        if street := query_dict.get("street"):
```
Maybe like this and then replace `query_dict["street"]` with `street` in line 83?